debian/calaos-server.service

```diff
@@ -1,9 +1,13 @@
 [Unit]
-Description=Calaos-Server Daemon
+Description=Calaos home automation server service
 After=network.target
 
 [Service]
-ExecStart=/usr/bin/calaos-server
+ExecStart=/usr/local/bin/calaos_server
```

Comment:
Also the .service file is in debian/ folder. This is not even working for building deb package, or is it?